app/app.js

```diff
@@ -8,7 +8,7 @@ require('angular-resource');
 require('angular-translate');
 require('angular-translate-loader-static-files');
 require('angular-ui-bootstrap');
-require('angular-datepicker');
+require('angular-datepicker/build/angular-datepicker');
```

Comment:
This breaks the app. I can only get it to work with `require('angular-datepicker/build/angular-datepicker');`

![image](https://cloud.githubusercontent.com/assets/1033332/15621006/ea87725c-242b-11e6-8df3-fc42ade24a70.png)